digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/VVHClient.scala

```diff
@@ -596,6 +660,16 @@ class VVHClient(vvhRestApiEndPoint: String) {
 
   }
 
+  protected def linkIdFromFeature(attributes: Map[String, Any]): Long = {
+    val linkId = attributes("LINKID").asInstanceOf[BigInt].longValue()
+    linkId
+  }
+
+  protected def extractLinkIdFromVVHFeature(feature: Map[String, Any]): Long = {
+    val attributes = extractFeatureAttributes(feature)
+    linkIdFromFeature(attributes)
```

Comment:
similar in this case, it's just a suggestion because the actual code it's not wrong:
linkIdFromFeature(extractFeatureAttributes(feature))